src/api/config.rs

```diff
@@ -58,6 +58,9 @@ pub struct EncoderConfig {
   // encoder configuration
   /// Video time base.
   pub time_base: Rational,
+  /// Flag specifying that the time base set in this configuration
+  /// should override any automatic time base detection.
+  pub time_base_override: bool,
```

Comment:
This seems a binary-only field, it should live in `rav1e.rs`.

Reply:
Not anymore with the last commit.

Reply:
It is still checked in the cli-only. Probably it could be done in a different way

Reply:
Ah okay

Reply:
Moved to `CliOptions`.